### PR TITLE
creating issue/dco check label text

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Created a follow-up text for the issue/dco label check.
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/label-commenter-config.yml [hunk 1]
```diff
@@ -16,6 +16,12 @@ labels:
       issue:
         body: Checking in... it has been awhile since we&#39;ve heard from you on this issue. Are you still working on it? Please let us know and please don&#39;t hesitate to contact a [MeshMate](https://layer5.io/community/meshmates/) or any other [community member](https://layer5.io/community/members) for assistance.
         action: open
+  - name: issue/dco
+    labeled:
+      issue:
```
Comment (alphaX86) [inline .github/label-commenter-config.yml:21]:
```suggestion
      pr:
```

**Review by alphaX86 — CHANGES_REQUESTED** [review]
From the comment given, I hope this would be applicable only to a PR and not issue

**Comment by alphaX86** [comment]
DCO 🙂🙂

**Review by alphaX86 — DISMISSED** [review]
LGTM on my side! 🎉

**Review by leecalcote — APPROVED** [review]
Thank you, @Anita-ihuman

**Pull request merged**